packages/at_follows_flutter/example/pubspec.yaml

```diff
@@ -37,6 +37,9 @@ dev_dependencies:
 dependency_overrides:
   at_onboarding_flutter:
     path: ../../at_onboarding_flutter
+  package_info_plus: ^3.0.0
+  win32: ^5.0.2
+  file_picker: ^5.3.2
```

Comment:
Instead of using `fle_picker` override, we can use onboarding: 6.0.0 to use latest file_picker.